app/static/app/js/components/Paginated.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import update from 'react-addons-update';
```

Comment:
From https://facebook.github.io/react/docs/update.html:

```
Note: update is a legacy add-on. Use kolodny/immutability-helper instead.
```

(I've never used either of these, which is why I went to go look it up.)